e2e_playwright/st_link_button.py

```diff
@@ -41,3 +41,17 @@
     use_container_width=True,
     help="help text here",
 )
+
+st.link_button(
+    "Primary with emoji icon",
+    "https://streamlit.io",
+    type="primary",
+    icon="🎈",
+)
+
+st.link_button(
+    "Primary with Material icon",
+    "https://streamlit.io",
+    type="primary",
+    icon=":material/bolt:",
+)
```

Comment:
Nit: would prefer no `type` param so that snapshots test the default button `type`

Reply:
Sounds good! Will update and then update the screenshots after the CI runs